### ISSUE TITLE
offline swagger/scalar ui

### ISSUE THREAD

**Issue by dargmuesli** [nitrojs/nitro#1757]
### Describe the feature

Elevating https://github.com/unjs/nitro/pull/1162/files#r1173842179 to an issue:

It would be great if `/_nitro/swagger` could make use of a locally available `swagger-ui-dist` instead of pulling it from `jsdeliver.net`.

### Additional information

- [ ] Would you be willing to help implement this feature?

**Comment by pi0** [comment]
Makes sense to allow offline support i am wondering if we can support it via an optional dependency.

Also it is not definite but we might replace swagger UI with something from unjs ;)

**Comment by tobychidi** [comment]
Yes. I would love something from UnJS

**Comment by pi0** [comment]
pending this idea for now as we need to find a better solution that has no major install size issues and works with scalar endpoint too.

**Comment by pi0** [comment]
Moving to https://github.com/unjs/openapi-renderer/issues/4 as nitro plans to eventually also migrate to this new library.
